Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView_Tests.swift

```diff
@@ -10,27 +10,39 @@ import XCTest
 class CurrentChatUserAvatarView_Tests: XCTestCase {
     var defaultAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
     var customAvatarView: CustomCurrentChatUserAvatarView! = CustomCurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+    var defaultAppearanceHookAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+
     var mockedCurrentUserController: CurrentChatUserController_Mock<NoExtraData>! = .mock()
 
-//    test_defaultAppearance() ✅
-//    test_contentIsLoaded() // klidne vic casu, treba "s" a "bez" online indicator
-//    test_customDefaultAppearance()
-//    test_customSubclassing() ✅
+    override func setUp() {
+        super.setUp()
+        UIConfig.default.currentUser.currentUserViewAvatarView.defaultAppearance.rules.removeAll()
+    }
```

Comment:
This is because of the appearance hooks tests? Maybe it will be easier to do something like:
```
class AppearanceHookTestAvatarView: CurrentChatUserAvatarView {}
AppearanceHookTestAvatarView.defaultApperance.addRule { ... }
```
so we don't have to mess with the global config

Reply:
I think resetting the config is no longer needed as I can do this in a way as you suggested with class at function scope 1 comment below.